### ISSUE TITLE
give higher parallelism or overclocking to electric implosion compressor 

### ISSUE THREAD

**Issue by SafariXiu** [GTNewHorizons/GT-New-Horizons-Modpack#12733]
### Your GTNH Discord Username

鄂木斯克间谍#9569

### Your Pack Version

2.2.10

### Your Proposal

Enhanced electric implosion compressor to give him higher parallelism or overclocking

加强电动聚爆压缩机，让他拥有更高的并行或者超频

### Your Goal

In the new version of the electric implosion compressor added many recipes that require a lot of synthesis, but the electric poly-explosion compressor can not be processed in parallel, which led to the whole process of crafting was pointlessly slowed down

在新版本中电动聚爆压缩机添加了许多需要大量合成的配方，但是电动聚爆压缩机无法并行处理，这导致整个制作过程被无意义的放慢了

### Your Vision

Reduce the meaningless waiting time, we should focus on more important and interesting things

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Connor-Colenso** [comment]
Do you mean the electric implosion compressor?

**Comment by Glease** [comment]
Issue edited to correct machine translation errors.

**Comment by Connor-Colenso** [comment]
Could probably add batch mode to the multi tbf.

**Comment by SafariXiu** [comment]
> Could probably add batch mode to the multi tbf.

But batch processing mode does not increase the efficiency of the machine, right? This function is used to reduce the delay of the machine, the output of items within a certain time is still the same

**Comment by miaowwwwww** [comment]
> 

but if it has batch mode, u can just parallel as many as u want. uiv 4 umv 16

**Comment by chochem** [comment]
duplicate of https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12071
